README.md

```diff
@@ -104,18 +105,18 @@ Setup configuration options:
 9. The default command prefix is &. If this is already in use on your server or you would like another prefix, you can change the `prefix` value in `config.json`.
 
 10. To configure lavalink:
-   * **If you are using Docker,** Open up Docker Desktop and find the lavalink container's name. Change the host IP listed in `config.json` to that name. For example, in the following image below, the config.json file should say `"host": "dozerRecent_lavalink_1"`. Set the `port` value to the port that's listed in `docker-compose.yml`.
+11. **If you are using Docker,** Open up Docker Desktop and find the lavalink container's name. Change the host IP listed in `config.json` to that name. For example, in the following image below, the config.json file should say `"host": "dozerRecent_lavalink_1"`. Set the `port` value to the port that's listed in `docker-compose.yml`.
    
    ![Finding the lavalink container name](static/containerNames.png)
 
-   * **If you are not using Docker**, set the `host` and `port` values to which values that you have set up.
+12. **If you are not using Docker**, set the `host` and `port` values to which values that you have set up.
```

Comment:
This as well as the below change are incorrect. They are branches of the same step, not separate steps. Please fix the numbering.

Reply:
Did it.